### ISSUE TITLE
How to set Y-Axis to integers?

### ISSUE THREAD

**Issue by floppyfish212** [ConsoleTVs/Charts#433]
Hi all, currently the Y-axis is set to decimal.

![screenshot_2018-10-26_20-20-57](https://user-images.githubusercontent.com/37132576/47565980-cd563e00-d95c-11e8-9617-344baf4d2182.png)

 How can i change that to just integers? I've gone through the code but didn't see any options to change that. Have I missed out something?

**Comment by ConsoleTVs** [comment]
I see you're using chart.js. Check their options and pass it via ->options([...]);

**Comment by dhabekar-praful** [comment]
`$chart->options([
                    'yAxis' => [
                        'enabled' => false
                    ]
                ]);  `
After using option I am getting following error:
`Call to undefined method ConsoleTVs\Charts\Builder\Chart::options()`

**Comment by ConsoleTVs** [comment]
You're not using the current version

**Comment by JDamour** [comment]
Is there any answer? I am using 6.5.4 version

**Comment by rafaelbaiolim** [comment]
@JDamour I don't know if you solved your problem, but this is how i'solved that on version 6.5.4:
```php

$chart->dataset("Acessos por mês do ano de {$this->ano}", 'line', $totMes)->options([
   'fill' => 'true',
   'borderColor' => '#51C1C0',
 ]);
        
$chart->options(
   ['scales' => [
      'yAxes' => [
          [
             'ticks' => [
                  'suggestedMax' => 5,
                   'beginAtZero' => true,
                   'stepSize' => 1
               ],
            ]
         ],
]]);
```
Result:
![image](https://user-images.githubusercontent.com/8842898/116510387-33f4a380-a89b-11eb-8034-d538ec9c3ad7.png)


The "stepSize" parameter will round the decimals. But I found useful "suggestedMax" when you have a small sized amount of data.

I had the same problem, because I' was using the 'scales' options like: `$chart->dataset()->options(..['scales'..])` instead of doing this, just call `$chart->options(['scales' => [...]])`

I'll let the last available documentation version of charJs v2 here, if someone need it:

<a href="https://www.chartjs.org/docs/2.9.4/axes/cartesian/linear.html#axis-range-settings">https://www.chartjs.org/docs/2.9.4/axes/cartesian/linear.html#axis-range-settings</a>

Hope It's helps someone !
